app/index.js

```diff
@@ -25,11 +25,6 @@ const defaultCamelVersion = "2.18.1";
 const defaultCamelDSL = "spring";
 const defaultPackagePrefix = "com.";
 
-var appname;
-var camelVersion;
-var camelDSL;
-var package;
```

Comment:
the variables are not used in the code?
seems strange that Sonar didn't reported unused variables but maybe it is due to the fact that javascript is not typed and so not possible to determine?